Packs/SailPointIdentityNow/Integrations/SailPointIdentityNow/command_examples

```diff
@@ -0,0 +1,24 @@
+!identitynow-search-identities query=id:2c918084740346d5017408d79229489e
```

Comment:
Please make sure there are no sensitive data here and general in the pack

Reply:
All the data was masked. No sensitive data committed.